<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lobbyName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>711</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Game Lobby</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lobbyStatus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Waiting for players...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="player5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jerry
Feng</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mimi
Jiao</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startGame">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>560</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>170</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anthony
Chen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>William
Yuan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jerry
Zhang</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Brian
Yang</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Albert
Yang</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player7">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>420</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kurtis
Carsch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leaveGame">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>560</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Leave Game</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
